l10n_it_ddt/data/ddt_data.xml

```diff
@@ -19,50 +19,50 @@
 
     <!-- carriage conditions -->
     <record id="carriage_condition_PF" model="stock.picking.carriage_condition">
-        <field name="name">PORTO FRANCO</field>
+        <field name="name">Carriage Paid</field>
     </record>
     <record id="carriage_condition_PA" model="stock.picking.carriage_condition">
-        <field name="name">PORTO ASSEGNATO</field>
+        <field name="name">Carriage Collect</field>
     </record>
     <record id="carriage_condition_PAF" model="stock.picking.carriage_condition">
-        <field name="name">CON ADDEBITO IN FATTURA</field>
+        <field name="name">Charged in Invoice</field>
     </record>
 
     <!-- goods descriptions -->
     <record id="goods_description_CAR" model="stock.picking.goods_description">
-        <field name="name">CARTONE</field>
+        <field name="name">Box</field>
     </record>
     <record id="goods_description_BAN" model="stock.picking.goods_description">
-        <field name="name">BANCALE</field>
+        <field name="name">Pallet</field>
     </record>
     <record id="goods_description_SFU" model="stock.picking.goods_description">
-        <field name="name">SFUSI</field>
+        <field name="name">Bulk</field>
     </record>
     <record id="goods_description_CBA" model="stock.picking.goods_description">
-        <field name="name">CARTONI-BANCALI</field>
+        <field name="name">Box-Pallet</field>
     </record>
 
     <!-- trasportation reasons -->
     <record id="transportation_reason_VEN" model="stock.picking.transportation_reason">
-        <field name="name">VENDITA</field>
+        <field name="name">Sale</field>
         <field name="to_be_invoiced" eval="True"/>
     </record>
     <record id="transportation_reason_VIS" model="stock.picking.transportation_reason">
-        <field name="name">CONTO VISIONE</field>
+        <field name="name">Goods on approval</field>
     </record>
     <record id="transportation_reason_RES" model="stock.picking.transportation_reason">
-        <field name="name">RESO</field>
+        <field name="name">Returned</field>
```

Comment:
Credo che in questo caso il termine RESO stia a significare un nome e non un aggettivo. Quindi in inglese si traduce con _Refund_ (come nome appunto), mentre _Returned_  è paragonabile a "restituito" (aggettivo).